collector/exchange.go

```diff
@@ -82,10 +82,17 @@ var (
 		"RpcClientAccess",
 	}
 
+	collectorWhitelist = make([]string, 0, len(exchangeAllCollectorNames))
```

Comment:
Could we make this a field on the `exchangeCollector` type? If not, it at least needs a more exchange-specific name, since it is a package level var.